### ISSUE TITLE
Parallel Port mod for 1541

### ISSUE THREAD

**Issue by mamejay** [tebl/C64-XUM1541#1]
Hello there.  I have made one of these and works perfectly.
I want to add a Parallel port to my 1541 to be able to connect to this XUM1541.
Can you please provide me pinout required on the 6522 chip?  Or is it just the I match up the PA0-7 with the D0-7 on this XUM1541?
Thanks

**Comment by tebl** [comment]
Yes, those are the one that need connecting. Note that some drives have pa0
shorted to ground, this needs to be fixed - either by lifting the pin from
a socket, or by cutting the link. You may need to check the internet for
more specific details as I've only owned one disk drive.

søn. 26. jun. 2022, 10:30 skrev mamejay ***@***.***>:

**Comment by mamejay** [comment]
Perfect!  I did find 64Copy Central site which has much more details regarding connecting to the drive.  I will doctor up a socket and do some testing.
